fusion_docs/guide/snapshot.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Snapshot for AWS Batch
+description: "Overview of the Fusion Snapshot feature for AWS Batch"
+date: "23 Aug 2024"
+tags: [fusion, storage, compute, file system, snapshot]
+---
+
+Fusion Snapshot enables you to create a checkpoint of a running Nextflow process and restore it on a different machine. Leveraging the Fusion file system, Snapshot enables you to move the task between different instances connected to the same S3 bucket.
+
+More specifically, the first use case for this feature is for Nextflow users to leverage AWS Spot instances without restarting an entire task from scratch if an instance is terminated. When a Spot instance interruption occurs, the task is restored from the last checkpoint on a new AWS instance, saving time and computational resources.
+
+Seqera Platform AWS Batch compute environments simplify the configuration and use of Fusion and Fusion Snapshot.
+
+### Seqera Platform compute environment requirements
+
+Fusion Snapshot v1.0.0 requires the following [Seqera compute environment](https://docs.seqera.io/platform/latest/compute-envs/aws-batch) configuration:
+
+- **Provider**: AWS Batch
+- **Pipeline work directory**: An S3 bucket in the same region as the compute environment
+- **Enable Wave containers**
+- **Enable Fusion v2**
+- **Enable fast instance storage**
+- **Config mode**: Batch Forge (recommended)
```

Comment:
recommended here means that is recommended for Fusion Snapshots or that is the general recommendation?

Because at the moment Fusion Snapshots are working in that mode exclusively.